### PR TITLE
Incrementally improve test coverage

### PR DESCRIPTION
**Pull request by mgasner**
Bumps coverage from 87.9% to 89.3% -- I'd like to get coverage above the 90% mark for 0.4.0

### CODE REVIEW

File: python_modules/dagster/dagster/seven/__init__.py [hunk 1]
```diff
@@ -71,3 +73,11 @@ def is_ascii(str_):
             return False
     else:
         return str_.isascii()
+
+
+if sys.version_info.major >= 3 and sys.version_info.minor >= 3:
```
Comment (schrockn) [inline python_modules/dagster/dagster/seven/__init__.py]: jesus

File: python_modules/dagster/dagster_tests/core_tests/definitions_tests/test_definitions.py [hunk 2]
```diff
@@ -25,6 +27,71 @@ def test_deps_equal():
...
+
+    assert solid_one_solid.has_input('input_one')
+
+    assert isinstance(solid_one_solid.input_def_named('input_one'), InputDefinition)
+
+    assert len(solid_one_solid.input_defs) == 1
+    assert len(solid_one_solid.output_defs) == 1
+
+    # These aren't python version independent because dict orders are different
+    # assert str(solid_one_solid.input_handle('input_one')) == (
```
Comment (schrockn) [inline python_modules/dagster/dagster_tests/core_tests/definitions_tests/test_definitions.py]: ?

File: python_modules/dagster/dagster_tests/core_tests/test_object_store.py [hunk 3]
```diff
@@ -313,6 +318,30 @@ def test_file_system_object_store_with_composite_type_storage_plugin():
                 ['hello'], context, resolve_to_runtime_type(List_(String_)), ['obj_name']
             )
 
+    with yield_pipeline_execution_context(
```
Comment (schrockn) [inline python_modules/dagster/dagster_tests/core_tests/test_object_store.py]: can you make an issue to make a generic wrapper around this

Comment (mgasner) [reply python_modules/dagster/dagster_tests/core_tests/test_object_store.py]: #1231

File: python_modules/dagit/dagit/app.py [hunk 4]
```diff
@@ -127,6 +130,7 @@ def open_file_view():
         return "Success", 200
 
     open_cmd = 'open' if sys.platform.startswith('darwin') else 'xdg-open'
+    # FIXME this doesn't work on python27
```
[the remark this answers is not in the document]
Comment (mgasner) [reply python_modules/dagit/dagit/app.py]: yep

File: python_modules/dagster/dagster/seven/__init__.py (hunk 1 above)
Comment (mgasner) [reply python_modules/dagster/dagster/seven/__init__.py]: lol

File: python_modules/dagster/dagster_tests/core_tests/definitions_tests/test_definitions.py (hunk 2 above)
Comment (mgasner) [reply python_modules/dagster/dagster_tests/core_tests/definitions_tests/test_definitions.py]: I can actually probably fix this by changing the way we generate the reprs

**Pull request merged**